model-optimizer/extensions/front/InterpolateV1ToInterpolateV4.py

```diff
@@ -0,0 +1,66 @@
+"""
+ Copyright (C) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from extensions.ops.interpolate import Interpolate
+from mo.front.common.partial_infer.utils import int64_array
+from mo.front.common.replacement import FrontReplacementSubgraph
+from mo.graph.graph import Graph
+from mo.ops.const import Const
+
+
+def correct_pad(pad):
+    return int64_array([pad] if not isinstance(pad, list) else pad)
+
+
+class InterpolateV1ToInterpolateV4(FrontReplacementSubgraph):
+    """
+    This transformation replaces the layer Interpolate-1 with the layer Interpolate-3.
```

Comment:
Please, use `operation` instead of `layer`

Reply:
Fixed.